File: java/app/src/main/java/com/ffo/mvp_java/mvp/model/DemoModel.java
package com.ffo.mvp_java.mvp.model;

import com.ffo.mvp_java.mvp.contract.DemoContract;

/**
 * @author: huchunhua
 * @create_time: 2018/5/2
 * @change_time:
 * @package: com.ffo.mvp_java.mvp.model
 * @project: java
 * @mail: huchunhua5@wasu.com
 * @describe: 一句话描述
 */
public class DemoModel implements DemoContract.Model{
    @Override
    public void demoModel(ModelListener modelListener) {
        modelListener.completed("mvp test");
    }
}

File: kotlin/app/src/main/java/pffo/com/mvp/medel/DemoModel.kt
package com.ffo.ipet.mvp.medel

import com.ffo.ipet.mvp.IModel
import com.ffo.ipet.mvp.contract.DemoContract

/**
 * @author: huchunhua
 * @create_time: 2018/4/25
 * @change_time:
 * @package: com.ffo.ipet.mvp.imp
 * @project: pet
 * @mail: huchunhua5@wasu.com
 * @describe: 一句话描述
 */
class DemoModel : IModel, DemoContract.IModel {

    override fun operationData(operationDataListener: DemoContract.IModel.OperationDataListener) {
        operationDataListener.completed("success")
    }

}